x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/analysis/analyzer/Verifier.java

```diff
@@ -87,9 +89,11 @@
  */
 public final class Verifier {
     private final Metrics metrics;
+    private final SqlConfiguration config;
```

Comment:
If only the version is needed, it would be easier to pass it directly instead of passing the Config object.

Reply:
I've changed it to only pass the Version.